### PR TITLE
Feat/callout

### PR DESCRIPTION
**Pull request by andysherman2121**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/1860

### Description

The Callout sub-pattern was created using the design and functional specs

Screen Shot 2020-04-30 at 4 43 40 PM

### CODE REVIEW

File: packages/react/src/patterns/sub-patterns/Callout/Callout.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+const Callout = ({ children }) => (
+  <section
+    className={`${prefix}--callout__container`}
+    data-autoid={`${stablePrefix}--callout__container`}>
+    <div
+      className={`${prefix}--callout__column`}
+      data-autoid={`${stablePrefix}--callout__column`}>
+      <div
+        className={`${prefix}--callout__object`}
```
Comment (annawen1) [inline packages/react/src/patterns/sub-patterns/Callout/Callout.js:29]: maybe we can call it `content` instead of `object`?

Comment (andysherman2121) [reply packages/react/src/patterns/sub-patterns/Callout/Callout.js:29]: yeah good idea

Comment (andysherman2121) [reply packages/react/src/patterns/sub-patterns/Callout/Callout.js]: fixed

File: packages/react/src/patterns/sub-patterns/Callout/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import 'carbon/ibmdotcom-styles/scss/patterns/sub-patterns/callout';
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Callout/README.md:13]: minor thing, forgot the `@`:
```
@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/callout/callout';
```

Comment (andysherman2121) [reply packages/react/src/patterns/sub-patterns/Callout/README.md:13]: good catch

File: packages/react/src/patterns/sub-patterns/Callout/__stories__/Callout.stories.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Sub-Patterns)|Callout', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('With Link List example', () => {
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Callout/__stories__/Callout.stories.js:16]: looked at the example and LinkList in this looks weird, I'm wondering if we should pick another example? Perhaps something that has a dark theme associated with it? Or maybe just a button?

File: packages/react/src/patterns/sub-patterns/Callout/__stories__/Callout.stories.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          <div className="bx--col-sm-4 bx--col-lg-8 bx--col-lg-12 bx--offset-lg-4">
+            <Callout children={children} />
+          </div>
+        </div>
+      </div>
+    );
+  })
+
+  .add('Simple example', () => {
+    const children = <p>hello world</p>;
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Callout/__stories__/Callout.stories.js:62]: this example doesn't have the hello world in the dark theme and looks off, maybe can manually set the hello world with the dark theme to it?

File: packages/react/src/patterns/sub-patterns/Callout/README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Callout } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const children = <p>hello world</p>;
+
+  return <Callout children={children} />;
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Callout/README.md:29]: sorry, one minor thing, we should probably document it as this:
```
return (
    <Callout>
       <p>hello world</p>
    </Callout>
);
```

**Pull request merged**